elide-standalone/src/main/java/com/yahoo/elide/standalone/config/ElideResourceConfig.java

```diff
@@ -110,27 +124,45 @@ protected void configure() {
 
         // Bind swaggers to given endpoint
         register(new org.glassfish.hk2.utilities.binding.AbstractBinder() {
+            @SuppressWarnings("rawtypes")
             @Override
             protected void configure() {
                 Map<String, Swagger> swaggerDocs = settings.enableSwagger();
                 if (!swaggerDocs.isEmpty()) {
                     // Include the async models in swagger docs
-                    if(settings.enableAsync()) {
-                        EntityDictionary dictionary = new EntityDictionary(new HashMap());
+                    if (settings.enableAsync()) {
+                        EntityDictionary dictionary = new EntityDictionary(new HashMap<>());
                         dictionary.bindEntity(AsyncQuery.class);
                         dictionary.bindEntity(AsyncQueryResult.class);
-                         
+
                         Info info = new Info().title("Async Service").version("1.0");
 
                         SwaggerBuilder builder = new SwaggerBuilder(dictionary, info);
-                        
+
                         //Default value of getJsonApiPathSpec() ends with /* at the end. need to remove.
                         String asyncBasePath = settings.getJsonApiPathSpec().replaceAll("/\\*", "");
 
                         Swagger swagger = builder.build().basePath(asyncBasePath);
 
                         swaggerDocs.put("async", swagger);
                     }
+                    // bind dynamic models
+                    if (settings.enableDynamicModelConfig()) {
+                        EntityDictionary dictionary = new EntityDictionary(new HashMap<>());
+                        for (Class entity : dynamicEntityCompiler.getBindClasses()) {
+                            dictionary.bindEntity(entity);
+                        }
+
+                        Info info = new Info().title("Dynamic models Service").version("1.0");
+                        SwaggerBuilder builder = new SwaggerBuilder(dictionary, info);
+
+                        //Default value of getJsonApiPathSpec() ends with /* at the end. need to remove.
+                        String dynamicBasePath = settings.getJsonApiPathSpec().replaceAll("/\\*", "");
+
+                        Swagger swagger = builder.build().basePath(dynamicBasePath);
+
+                        swaggerDocs.put("dynamic model", swagger);
```

Comment:
lets have a singular name instead of with spaces. Just "dynamic" instead of "dynamic model";